CallAutomation_OutboundCalling/README.md

```diff
@@ -0,0 +1,53 @@
+|page_type|languages|products
+|---|---|---|
+|sample|<table><tr><td>Typescript</tr></td></table>|<table><tr><td>azure</td><td>azure-communication-services</td></tr></table>|
+
+# Call Automation - Quick Start Sample
+
+This sample application shows how the Azure Communication Services  - Call Automation SDK can be used to build IVR related solutions. 
+It makes an outbound call to a phone number, performs DTMF recognition, plays a different audio message based on the key pressed by the callee and hangs-up the call. 
+This sample application configured for accepting tone 1 (tone1), 2 (tone2) , If the callee pressed any other key than expected, the call will be disconnected.
+This sample application is also capable of making multiple concurrent outbound calls. The application is a web-based application built on Express.js framework.
+
+# Design
+
+![design](./data/OutboundCallDesign.png)
+
+## Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/)
+- [Visual Studio (2022 v17.4.0 and above)](https://visualstudio.microsoft.com/vs/)
```

Comment:
Do we need to use Visual Studio or Visual Studio code

Reply:
code, will upddate